### ISSUE TITLE
Plugin not working at all, neovim v0.4.3

### ISSUE THREAD

**Issue by hatimmohammed369** [adi/vim-indent-rainbow#2]
I'm sorry but this plugin is refusing to work
Here's what I do:
$ git clone git@github.com:adi/vim-indent-rainbow.git
$ cp vim-indent-rainbow/autoload/* ~/.vim/autoload
Then I add
call togglerb#map("<F9>")
at the top of ~/.vimrc
I open Neovim, hit F9 but nothing happens

**Comment by adi** [comment]
I haven't tested it neovim but will do and see what's wrong.

**Comment by bakerbrandond** [comment]
bump

**Comment by brandonrobertz** [comment]
yeah this doesn't load at all, the `rainbow#enable()` doesn't appear to even be registered.
